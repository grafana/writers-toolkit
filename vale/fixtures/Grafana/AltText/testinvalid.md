Don't use:

- ![](/path/to/image/)
