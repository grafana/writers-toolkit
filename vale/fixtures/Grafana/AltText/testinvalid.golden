testinvalid.md:3:3:Grafana.AltText:All images must have alt text.
